src/PinAssignment/PackagePinsLoader.cpp

```diff
@@ -43,23 +44,24 @@ std::pair<bool, QString> PackagePinsLoader::load(const QString &fileName) {
 #endif
   lines.pop_front();  // header
   PackagePinGroup group{};
+  QSet<QString> uniquePins;
   for (const auto &line : lines) {
```

Comment:
Not necessary for this check in, but code blocks like these could benefit from comments. The interaction between the different model, groups, etc isn't obvious if you aren't already familiar with the code.

Reply:
agree